AssetManagementTestProject/PageObj/HomePage.cs

```diff
@@ -6,23 +6,18 @@ public class HomePage : WebDriverAction
     private string btnNavigationBar = "//div[contains(@class, 'ant-dropdown-trigger cursor-pointer')]";
     private string btnChangePw = "//a[contains(@href, '/change-password')]";
     private string btnLogout = "//a[contains(@href, '/logout')]";
-    private string headerHomePage = "//h1[contains(@class, 'text-red-600')]";
+    public string HeaderHomePage = "//h1[contains(@class, 'text-red-600')]";
+    
+    #region MY ASSIGNMENT LIST
+    public string HeaderMyAssignment = "//h1[text()='My Assignment']";
+    public string BtnAcceptAssignment = "(//span[contains(@aria-label, 'check')])[1]";
+    public string BtnDeclineAssignment = "(//span[contains(@aria-label, 'close')])[1]";
+    public string BtnRequestReturnAsset = "(//span[contains(@aria-label, 'undo')])[1]";
 
```

Comment:
I added it a while ago, because without it the test cannot run

![image](https://user-images.githubusercontent.com/79431746/206141033-62b1e10a-fdfc-4a49-97c5-f58443b74eb1.png)